tests/test_cluster/test_elbow.py

```diff
@@ -184,11 +184,31 @@ def test_invalid_k(self):
         """
 
         with pytest.raises(YellowbrickValueError):
-            KElbowVisualizer(KMeans(), k=(1,2,3,4,5))
+            KElbowVisualizer(KMeans(), k=(1, 2, 3, 'foo', 5))
 
         with pytest.raises(YellowbrickValueError):
             KElbowVisualizer(KMeans(), k="foo")
 
+    def test_valid_k(self):
+        """
+        Assert that valid values of K generate correct k_values_:
+        if k is an int, k_values_ = range(2, k+1)
+        if k is a tuple of 2 ints, k_values = range(k[0], k[1])
+        if k is an iterable, k_values_ = list(k)
+        """
+        visualizer = KElbowVisualizer(KMeans(), k=8)
+        assert visualizer.k_values_ == list(np.arange(2, 8+1))
+
+        visualizer = KElbowVisualizer(KMeans(), k=(4, 12))
+        assert visualizer.k_values_ == list(np.arange(4, 12))
+
+        visualizer = KElbowVisualizer(KMeans(), k=np.arange(10, 100, 10))
+        assert visualizer.k_values_ == list(np.arange(10, 100, 10))
+
+        visualizer = KElbowVisualizer(KMeans(),
+                                      k=[10, 20, 30, 40, 50, 60, 70, 80, 90])
+        assert visualizer.k_values_ == list(np.arange(10, 100, 10))
+
```

Comment:
- Great tests... I believe you covered all inputs types.  
- I'm wondering if you might want to test that a `YellowbrickValueError` is raised when non-integer values are input into `k`.  @bbengfort and @rebeccabilbro can you chime in and let us know if this extra test is necessary?

Reply:
Hey there @lwgray - by non-integer are you thinking floats? Looks like @drwaterman already has test cases for strings.